src/node/desktop/src/main/application.ts

```diff
@@ -136,7 +136,8 @@ export class Application implements AppState {
       }
     }
 
-    if (!await prepareEnvironment()) {
+    const [, error] = prepareEnvironment();
```

Comment:
As mentioned in Slack, suggest changing `prepareEnvironment` to just return `Err` since we never use the result.